### ISSUE TITLE
Test Edge Maintenance Releases

### ISSUE THREAD

**Issue by phet-steele** [phetsims/tasks#523]
Found here: http://www.colorado.edu/physics/phet/dev/olsonjb/maintenance/

Look specifically for visual issues in Edge, and that the PhET menu and NavBar behave nicely.

**Comment by phet-steele** [comment]
@jonathanolson @ariel-phet done.

**Comment by jonathanolson** [comment]
New batch of maintenance-release candidates at http://www.colorado.edu/physics/phet/dev/olsonjb/maintenance2/

Recommend brief testing + checks for the "full screen is broken" (bad resizing, NOT the crazier one), and the dashed-lines.

**Comment by ariel-phet** [comment]
Back at you @phet-steele , hopefully lots of these are fixed now.

**Comment by phet-steele** [comment]
@jonathanolson am I to assume the above Acid Base Solutions and Graphing Lines issues are resolved? Because they aren't.

**Comment by jonathanolson** [comment]
Those are not yet fixed. Should we wait on testing for those sims to fix those issues?

**Comment by phet-steele** [comment]
@jonathanolson yes, I can test everything but those two.

**Comment by jonathanolson** [comment]
http://www.colorado.edu/physics/phet/dev/olsonjb/maintenance3/graphing-lines_en.html should be good for testing now also (includes fix for https://github.com/phetsims/graphing-lines/issues/60).

**Comment by jonathanolson** [comment]
http://www.colorado.edu/physics/phet/dev/olsonjb/maintenance3/acid-base-solutions_en.html also should be good, includes a patch for https://github.com/phetsims/acid-base-solutions/issues/145.

Please keep https://github.com/phetsims/acid-base-solutions/issues/145 open, as it isn't fixed in master.

**Comment by jonathanolson** [comment]
Also https://github.com/phetsims/molecules-and-light/issues/132 and https://github.com/phetsims/joist/issues/322 are deferred, so I believe I've patched all the other issues (just pending testing now).

**Comment by phet-steele** [comment]
@jonathanolson all is well, testing done.

**Comment by jonathanolson** [comment]
Thanks! Will be deploying the maintenance releases.
